### PR TITLE
Added docker

### PR DESCRIPTION
**Pull request by GlebPashko**

### CODE REVIEW

**Review by sarakhmen — CHANGES_REQUESTED** [review]
please add minor improvement ;)

File: src/main/resources/application.properties [hunk 1]
```diff
@@ -1,11 +1,2 @@
 spring.application.name=book-store
-spring.datasource.url=jdbc:mysql://localhost:3306/booksdb?serverTimeZone=UTC
-spring.datasource.username=root
-spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver
-
-spring.jpa.hibernate.ddl-auto=validate
-spring.jpa.show-sql=true
 spring.jpa.open-in-view=false
```
Comment (liliia-ponomarenko) [inline src/main/resources/application.properties:2]: Better return it if you want to have the possibility to run your project locally

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+RUN java -Djarmode=layertools -jar application.jar extract
+
+FROM openjdk:17-jdk-alpine
+WORKDIR application
+COPY --from=builder application/dependencies/ ./
+COPY --from=builder application/spring-boot-loader/ ./
+COPY --from=builder application/snapshot-dependencies/ ./
+COPY --from=builder application/application/ ./
+ENTRYPOINT ["java", "org.springframework.boot.loader.launch.JarLauncher"]
+EXPOSE 8080
```
Comment (liliia-ponomarenko) [inline Dockerfile:14]: add one empty line to the end of the file ;)

File: docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+      SPRING_APPLICATION_JSON: '{
+        "spring.datasource.url" : "jdbc:mysql://mysqldb:$MYSQLDB_DOCKER_PORT/$MYSQLDB_DATABASE?characterEncoding=UTF-8&serverTimezone=UTC",
+        "spring.datasource.username" : "$MYSQLDB_USER",
+        "spring.datasource.password" : "$MYSQLDB_ROOT_PASSWORD",
+        "spring.jpa.hibernate.ddl-auto" : "validate",
+        "spring.jpa.show-sql" : "true",
+        "jwt.expiration" : "$JWT_EXPIRATION_TIME",
+        "jwt.secret" : "$JWT_SECRET"
+      }'
+      JAVA_TOOL_OPTIONS: "-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=*:$DEBUG_PORT"
```
Comment (liliia-ponomarenko) [inline docker-compose.yml:35]: the same

**Review by liliia-ponomarenko — APPROVED** [review]
Good job! A few comments. Also better add env.sample file ;)

**Pull request merged**